packages/web-components/src/accordion/accordion-item/accordion-item.styles.ts

```diff
@@ -42,20 +46,17 @@ export const accordionItemStyles = (context, definition) =>
         grid-column: 2;
         grid-row: 1;
         outline: none;
-        padding: 0 calc((6 + (var(--design-unit) * 2 * var(--density))) * 1px);
+        padding: 0 calc((6 + (${designUnit} * 2 * ${density})) * 1px);
         text-align: left;
         height: calc(${heightNumber} * 1px);
-        color: ${neutralForegroundRestBehavior.var};
+        color: ${neutralForegroundRest};
         cursor: pointer;
         font-family: inherit;
     }
 
-    .button:hover {
-        color: ${neutralForegroundHoverBehavior.var};
-    }
-
+    .button:hover,
     .button:active {
-        color: ${neutralForegroundActiveBehavior.var};
+        color: ${neutralForegroundRest};
```

Comment:
Was thi supposed to change from active to rest?

Reply:
@EisenbergEffect Per @nicholasrice there isn't really a difference between rest and hover, active states. This is intentional unless there is an update where these should resolve and return different values.

Reply:
Yea I'd defer to @bheston on this, and on whether we actually want stateful neutralForeground colors. We have them today but they all resolve to the same color, so I only added the 'rest' thinking we could expand in the future if necessary.

Reply:
I think this is okay for now, but if we do find a *need* where these wouldn't resolve to the same color we can add/update. Right now it seems like a fair amount of overhead to have multiple recipes that resolve to the same thing. I'm hopeful that we can take this as a follow-up if needed now, or we can add in the future as a feature.